keps/sig-node/2403-pod-resources-allocatable-resources/kep.yaml

```diff
@@ -26,13 +26,13 @@ stage: beta
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.23"
+latest-milestone: "v1.28"
```

Comment:
not sure if anybody using this field, just a small recommendation to update the `last-updated` above

Reply:
right, thanks for pointing out. I'll make sure the template is up to date.